scripts/test_all.wren

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env wrenc
+
+import "io" for Directory, File
+import "os" for Process
+import "meta" for Meta
+import "essentials" for Strings
+
+// 1. set cwd to wren-testie root
+
+// TODO (enhancement): static method File.tempfile()
+// TODO (enhancement): reading stdout of a process
+
+var tmpfile = ".git.root.%(Process.pid)"
+Process.exec("sh", ["-c", "git rev-parse --show-toplevel > %(tmpfile)"])
+var rootdir = File.read(tmpfile).trim()
+File.delete(tmpfile)
+
+Process.chdir(rootdir)
+
+import "../testie" for Testie
+import "../src/expect" for Expect
```

Comment:
```suggestion
import "../testie" for Testie, Expect
```

Testie should be the source for everything.

Reply:
Yes, right. I feel there was a reason I did that, but I don't remember it now.